### PR TITLE
WIP docs: Add initial documentation with goals

### PR DESCRIPTION
**Pull request by stefwalter**
Here's the initial documentation with the mission statement, links to related project, and description of how things come together.

### CODE REVIEW

**Comment by stefwalter** [comment]
> From CI/CD point of view, I would suggest to use the github.com/containers/containers.github.io repo instead. There are few benefits of using it. According to the [instructions](https://pages.github.com/) provided by github, using the suggested repo name provides out of the box CD pipelines which are maintained by GitHub.
> 
> And something I picked up with help from chatgpt:
> 
> Using GitHub's suggested repository name for a GitHub Pages site can offer several benefits:
> 
>     1. Convenience: GitHub's suggested repository name is usually based on the username and project name, making it easy to remember and relevant to your project. This saves you time and effort in coming up with a unique name yourself.
> 
>     2. Consistency: GitHub's suggested repository name typically follows their naming conventions, which can help maintain consistency across your projects and make them easier to identify within your GitHub profile.
> 
>     3. SEO (Search Engine Optimization): GitHub's suggested repository name might include keywords related to your project, which could potentially improve the searchability of your GitHub Pages site on search engines like Google.
> 
>     4. Professionalism: Using a name suggested by GitHub can lend a sense of professionalism to your project, especially if you're collaborating with others or showcasing your work to potential employers or clients.
> 
>     5. Community Recognition: If your GitHub Pages site gains traction within the GitHub community, having a recognizable and easily identifiable repository name can help others find and engage with your project more easily.
> 
> 
> Overall, while using GitHub's suggested repository name is not mandatory, it can be a practical choice for many GitHub Pages sites, offering convenience, consistency, and potential SEO benefits. However, it's always a good idea to ensure that the suggested name aligns well with your project's branding and objectives.

Could you discuss this one with @rhatdan I think he is one of the admins of the containers namespace on GitHub. I'm fine with moving it if we want it in the top level repository.

**Comment by stefwalter** [comment]
In order to get a GitHub pages started, I moved teh content directly into the main and closed this pull request.

File: docs/src/projects.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# What are the projects involved or related?
+
+The bootc tooling and project are the nexus of a lot of the functionality that enable containers to be used to deploy Linux systems:
+
+ * [bootc documentation](https://containers.github.io/bootc/)
+ * [bootc on Github](https://github.com/containers/bootc)
```
[the remark this answers is not in the document]
Comment (stefwalter) [reply docs/src/projects.md:6]: Couldn't Docker be used for those workflows as well?

One of the key goals an concepts of Bootable Containers is to use all the container build, registry and related standards and tooling. I understand we are all using podman, but docker should be equally usable for the fundamental workflows involved.

Thus I've kept Podman on the line with other contianer related tooling. 

Once we get to trust chain work, I think there may be scenarios where podman is necessary and docker is insufficent.

[the remark this answers is not in the document]
Comment (vrothberg) [reply docs/src/projects.md:6]: > Couldn't Docker be used for those workflows as well?

For building the images, absolutely.  Yet, bootc uses Podman as a core component _inside_ of those images.

**Pull request closed without merging**